src/api/app/controllers/source_controller.rb

```diff
@@ -1560,11 +1560,11 @@ def package_command_linktobranch
 
     # convert link to branch
     rev = ''
-    if not pkg_rev.nil? and not pkg_rev.empty?
+    if !pkg_rev.nil? && !pkg_rev.empty?
```

Comment:
Oh, you are right. 

@Ana06 @mdeniz Anyway, this PR is about changing and, or, not to &&, ||, !. I would recommend you two keep this focus and do additional changes in a follow up PR. Otherwise the discussions get very messy. Actually it already is;-)